src/Workleap.Extensions.Mongo/Indexing/MongoIndexer.cs

```diff
@@ -27,23 +30,22 @@ public Task UpdateIndexesAsync(Assembly assembly, string? clientName = null, str
         return this.UpdateIndexesAsync(new[] { assembly }, clientName, databaseName, cancellationToken);
     }
 
+    public Task UpdateIndexesAsync(string? clientName = null, string? databaseName = null, CancellationToken cancellationToken = default)
+    {
+        return this.UpdateIndexesAsync(Enumerable.Empty<Assembly>(), clientName, databaseName, cancellationToken);
```

Comment:
SHould we just call this method instead since it won't do anything with the list of assembly? 
![image](https://github.com/gsoft-inc/wl-extensions-mongo/assets/8277266/2cb7a3f9-63f6-4091-acd9-72576690390b)